### ISSUE TITLE
Composite Catalog Endpoint Only Showing First Source

### ISSUE THREAD

**Issue by jtwiest** [urbica/martin#652]
Using version 0.8.0

We're using composite sources and the catalog endpoint only seems to be listed the first vector layer vs. all of them. The tiles endpoint with the same composite sources does render all of the correct layers. Below you can see I'm requesting a composite of `tiles.layer_one` and `tiles.layer_two`, it's confirmed Martin sees both layers since they're listed in the `tiles` url in the response however the `vector_layers` array only lists the first composite layer `tiles.layer_one` vs. both.

```
curl localhost:3000/tiles.layer_one,tiles.layer_two

{
    "tilejson": "3.0.0",
    "tiles": [
        "http://localhost:3000/tiles.layer_one,tiles.layer_two/{z}/{x}/{y}"
    ],
    "vector_layers": [
        {
            "id": "tiles.layer_one",
            "fields": {
                ...
            }
        }
    ],
    "bounds": [
        ...
    ],
    "description": "tiles.layer_one.geom",
    "name": "tiles.layer_one"
}
```

When using mapbox this does actually appear to work fine, both layers one and two are available to the user. The downside however is that every time I try to add layer two as a layer to the map, I get the following error:

`Error: Source layer "tiles.layer_two" does not exist on source "my_source" as specified by style layer "layer_two"`

**Comment by nyurik** [comment]
@jtwiest I'm not sure I understood.  You ask for two "Tile Sources" - which is different from "MVT layers" -- each tile source can have more than one layers. Each MVT layer should have its own object in the `vector_layers` array. When asking for merged tile sources, `vector_layers` should in theory merge as well.  Are you saying it does not?

**Comment by nyurik** [comment]
I think i found the issue, fixed in #653

**Comment by jtwiest** [comment]
Looks like that fixed it! Thank you @nyurik

**Comment by nyurik** [comment]
sure, i still need to do a release (we have had some issues with the docker for mac, so may have to revert that part)

**Comment by nyurik** [comment]
released as 0.8.2
